CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # Change Log
 
+** Version 3.3.0 *(2020-09-30)*
+
+This is a new minor release with multiple fixes and improvements. 
+Note, this release is the last one with minSDK 16. In next versions Chucker will bump minSDK 21 in order to use newer OkHttp versions.
```

Comment:
```suggestion
Note, this release is the last one with minSDK 16. In next versions Chucker will bump minSDK to 21 in order to use newer OkHttp versions.
```

Reply:
I'm not sure about the strategy but is it really the last release? I thought there is some grace period for critical bug fixes and maybe help with compatibility if we'd like to change something in 4.x and add a migration to 3.x.

Reply:
Right, seems I put it wrong, considering our discussions in Slack. Let me update this statement according to https://kotlinlang.slack.com/archives/CRWD6370R/p1594367925085200